gallery/src/components/NxTable/NxTablePage.tsx

```diff
@@ -243,6 +245,13 @@ export default function NxTablePage() {
         An example with a sortable column.
       </GalleryExampleTile>
 
+      <GalleryExampleTile title="Filter Columns Example"
+                          id="nx-table-filter-example"
+                          liveExample={NxTableFilterExample}
+                          codeExamples={tableFilterExample}>
+        An example with filter columns.
+      </GalleryExampleTile>
```

Comment:
We have two table-related pages in the gallery: this one for the `NxTable` react component, and one for the raw `nx-table` styles.  Should an example of the filter styles, and documentation of the new CSS class, be added to the other page as well?  If so, I'd say the example on that page wouldn't need to be functional, just to have a static layout, perhaps of an empty table with a filter header row.  @glenhunter what do you think?

Reply:
> If so, I'd say the example on that page wouldn't need to be functional, just to have a static layout, perhaps of an empty table with a filter header row. @glenhunter what do you think?

I'd be happy with that, or even with just a note that to see examples of filtering (and sorting) you should look at the NxTable component page.

Reply:
Done: https://github.com/sonatype/sonatype-react-shared-components/pull/165/commits/bb849391441ee299fe6f2b9f5fb314ca5cc65106